server/flyway/sql/V20__fix_timestamp_to_timestamptz.sql

```diff
@@ -0,0 +1,80 @@
+
+ALTER TABLE app_fam.fam_role ALTER COLUMN create_date TYPE TIMESTAMP(6) WITH TIME ZONE;
+ALTER TABLE app_fam.fam_role ALTER COLUMN update_date TYPE TIMESTAMP(6) WITH TIME ZONE;
+
+ALTER TABLE app_fam.fam_role_type ALTER COLUMN effective_date TYPE TIMESTAMP(6) WITH TIME ZONE;
+ALTER TABLE app_fam.fam_role_type ALTER COLUMN expiry_date TYPE TIMESTAMP(6) WITH TIME ZONE;
+ALTER TABLE app_fam.fam_role_type ALTER COLUMN update_date TYPE TIMESTAMP(6) WITH TIME ZONE;
+
+ALTER TABLE app_fam.fam_user ALTER COLUMN create_date TYPE TIMESTAMP(6) WITH TIME ZONE;
+ALTER TABLE app_fam.fam_user ALTER COLUMN update_date TYPE TIMESTAMP(6) WITH TIME ZONE;
+
+ALTER TABLE app_fam.fam_user_group_xref ALTER COLUMN create_date TYPE TIMESTAMP(6) WITH TIME ZONE;
+ALTER TABLE app_fam.fam_user_group_xref ALTER COLUMN update_date TYPE TIMESTAMP(6) WITH TIME ZONE;
+
+ALTER TABLE app_fam.fam_user_role_xref ALTER COLUMN create_date TYPE TIMESTAMP(6) WITH TIME ZONE;
+ALTER TABLE app_fam.fam_user_role_xref ALTER COLUMN update_date TYPE TIMESTAMP(6) WITH TIME ZONE;
+
+ALTER TABLE app_fam.fam_user_type_code ALTER COLUMN effective_date TYPE TIMESTAMP(6) WITH TIME ZONE;
+ALTER TABLE app_fam.fam_user_type_code ALTER COLUMN expiry_date TYPE TIMESTAMP(6) WITH TIME ZONE;
+ALTER TABLE app_fam.fam_user_type_code ALTER COLUMN update_date TYPE TIMESTAMP(6) WITH TIME ZONE;
+
+
+-- Change default value for timestamptz column from localtimestamp to current_timestamp so it includes a time zone
+ALTER TABLE app_fam.fam_application ALTER COLUMN create_date SET DEFAULT CURRENT_TIMESTAMP;
+ALTER TABLE app_fam.fam_application ALTER COLUMN update_date SET DEFAULT CURRENT_TIMESTAMP;
+
+ALTER TABLE app_fam.fam_application_client ALTER COLUMN create_date SET DEFAULT CURRENT_TIMESTAMP;
+ALTER TABLE app_fam.fam_application_client ALTER COLUMN update_date SET DEFAULT CURRENT_TIMESTAMP;
+
+ALTER TABLE app_fam.fam_application_group_xref ALTER COLUMN create_date SET DEFAULT CURRENT_TIMESTAMP;
+ALTER TABLE app_fam.fam_application_group_xref ALTER COLUMN update_date SET DEFAULT CURRENT_TIMESTAMP;
+
+ALTER TABLE app_fam.fam_forest_client ALTER COLUMN create_date SET DEFAULT CURRENT_TIMESTAMP;
+ALTER TABLE app_fam.fam_forest_client ALTER COLUMN update_date SET DEFAULT CURRENT_TIMESTAMP;
+
+ALTER TABLE app_fam.fam_group ALTER COLUMN create_date SET DEFAULT CURRENT_TIMESTAMP;
+ALTER TABLE app_fam.fam_group ALTER COLUMN update_date SET DEFAULT CURRENT_TIMESTAMP;
+
+ALTER TABLE app_fam.fam_group_role_xref ALTER COLUMN create_date SET DEFAULT CURRENT_TIMESTAMP;
+ALTER TABLE app_fam.fam_group_role_xref ALTER COLUMN update_date SET DEFAULT CURRENT_TIMESTAMP;
+
+ALTER TABLE app_fam.fam_role ALTER COLUMN create_date SET DEFAULT CURRENT_TIMESTAMP;
+ALTER TABLE app_fam.fam_role ALTER COLUMN update_date SET DEFAULT CURRENT_TIMESTAMP;
+
+ALTER TABLE app_fam.fam_role_type ALTER COLUMN effective_date SET DEFAULT CURRENT_TIMESTAMP;
```

Comment:
Currently there is no default value for the effective_date in the fam_role_type table, but I saw other tables who use effective_date (fam_app_environment and fam_user_type_code) both have a default value, so I added a default value here as well

Reply:
I assume "effective_date" and "update_date" don't need default value as they will be updated by insert statement or application.

Reply:
Yes, I think we have update_date be null for newly created records, in which case there should be no default set for update columns.

Reply:
yeah, I would agree, but I saw V4, V10, and V18 already set the default value for update_date and effective_date to LOCALTIMESTAMP. Maybe here I should cleanup all the default value?

Reply:
Based on our conversation will keep it as now.